src/Database/DBA.php

```diff
@@ -582,7 +583,8 @@ public static function p($sql) {
 			self::$errorno = $errorno;
 		}
 
-		$a->saveTimestamp($stamp1, 'database');
+		/// @todo profiling out of App
+		//$a->saveTimestamp($stamp1, 'database');
```

Comment:
Is this PR effectively killing any DB profiling?

Reply:
I'm afraid.. Yes for now.. There's a loopback to the `App` class for profiling.

I started creating a `Profiler` class to encapsulate the `performance[]` and `callstack[]` information, but the PR is big enough.

If it's a showstopper, I can create a PR for the `Profiler` first.

@ananndo I think this is something important for you too.

Reply:
will be fixed per #6673